test/testwave/testwave_app.cpp

```diff
@@ -1009,11 +1041,14 @@ testwave_app::initialise_options(Context& ctx, po::variables_map const& vm,
         for (std::vector<std::string>::const_iterator cit = syspaths.begin();
               cit != end; ++cit)
         {
+            std::string full(*cit);
+            got_expected_result(ctx.get_current_filename(),"",full);
```

Comment:
I (ab)used the got_expected_results function to make the include paths into full paths via the $P placeholder.

Reply:
Fair enough.